### PR TITLE
DOC-4281: Add the Gamification chapter

### PR DESCRIPTION
**Pull request by hghorbel**

### CODE REVIEW

File: docs/Gamification.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
+.. _Gamification:
+
+####################
+Gamification
+####################
+
+Engaging your collaborators is one of the key challenges for organizations nowadays. 
+
+**eXo Gamification** will help your organizations to engage your employees.
```
Comment (mzorai) [inline docs/Gamification.rst]:
```suggestion
**eXo Gamification** will help your organization to engage your employees.
```

File: docs/Gamification.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+**eXo Gamification** is layering game-like features upon the use of applications –such as those used within a digital workplace– in order to boost employee engagement to enhance the use of your digital workplace solution by encouraging collaborators to participate.
```
Comment (mzorai) [inline docs/Gamification.rst]:
```suggestion
**eXo Gamification** is layering game-like features upon the use of applications, such as those used within a digital workplace, in order to boost employee engagement and enhance the use of your digital workplace solution by encouraging collaborators to participate.
```

File: docs/Gamification.rst [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+When your users are engaged, they participate more fully on your platform by posting , commenting, sharing documents, rating, posting, downloading, or performing other actions.
```
Comment (mzorai) [inline docs/Gamification.rst]:
```suggestion
When employees are engaged, they more participate on your platform by posting, commenting, sharing documents, rating, downloading and much more. 
```

**Review by mzorai — CHANGES_REQUESTED** [review]

File: docs/Gamification.rst [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+.. _Domains:
+
+=======================
+Domains
+=======================
+
+.. _Badges:
+
+=======================
+Domains
```
Comment (mkrout) [inline docs/Gamification.rst]: Should be badges

**Review by mkrout — CHANGES_REQUESTED** [review]

**Review by mzorai — APPROVED** [review]

**Review by mkrout — APPROVED** [review]

**Pull request merged**